oscafrica/osf-website
# made changes to the sponsorship deck document

#### What does this PR do
made changes to the sponsorship deck document

#### Description of Task to be completed

#### How should this be manually tested
Open the website, click on "Join our sponsors" and you should see the updated sponsor decks document

#### What are the relevant issues this PR belongs to

#<NUMBER>

#### Any background context you want to add

#### Screenshots
